Defs/Ammo/Other/Flamethrower.xml

```diff
@@ -148,6 +173,20 @@
 		</projectile>
 	</ThingDef>
 
+	<ThingDef ParentName="BaseFlamethrowerBullet" MayRequire="Ludeon.Rimworld.Anomaly">
+		<defName>Bullet_Flamethrower_Bioferrite</defName>
+		<thingClass>CombatExtended.ProjectileCE_Explosive</thingClass>
+		<label>infused chemfuel stream</label>
+		<projectile Class="CombatExtended.ProjectilePropertiesCE">
+			<damageDef>BioferriteFlame</damageDef>
+			<damageAmountBase>3</damageAmountBase>
```

Comment:
Burst  (currenlty 5 damage per) and regular (currently 3 damage per) should share damage and it should be 5 for both given that bioferrite is an exotic ammo, you need to harvest it from contained entities which is arguably at least a much of a risk as promethium is.